Microsoft.Research/ClousotMain/Caching/CacheManager.cs

```diff
@@ -1121,7 +1121,8 @@ private void InstallAssumes(InferredExpr<Field, Method> inferredExpr, Method met
 
                     this.ReplayOutput.WriteLine("The reason why the deserialization failed is {0}", e.Message);
                     this.ReplayOutput.WriteLine(
-                        "We remove the incorrect entry from the cache and we replace it with a new one. If the error persists, please report the tool output to logozzo@microsoft.com");
+                        "We remove the incorrect entry from the cache and we replace it with a new one. "
+                        + "If the error persists, please report the tool output at https://github.com/Microsoft/CodeContracts/issues");
```

Comment:
@SergeyTeplyakov I'm fairly certain we are going to change it *exactly* one more time, but it still could be a good idea to separate it. :+1: